diff --git a/autosuspend.py b/autosuspend
similarity index 100%
rename from autosuspend.py
rename to autosuspend
diff --git a/autosuspend.service b/autosuspend.service
index a5d58ed..2f55d1c 100644
--- a/autosuspend.service
+++ b/autosuspend.service
@@ -3,7 +3,7 @@ Description=A daemon to suspend your server in case of inactivity
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/autosuspend.py -l /etc/autosuspend-logging.conf
+ExecStart=/usr/bin/autosuspend -l /etc/autosuspend-logging.conf
 
 [Install]
 WantedBy=multi-user.target
diff --git a/setup.py b/setup.py
index 0cd3dce..adf60e1 100644
--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,7 @@ setup(
     ],
 
     scripts=[
-        'autosuspend.py'
+        'autosuspend'
     ],
     data_files=[
         ('etc', ['autosuspend.conf',
